mkdocs/commands/gh_deploy.py

```diff
@@ -126,6 +132,10 @@ def gh_deploy(config, message=None, force=False, ignore_version=False):
         if host is None:
             # This could be a GitHub Enterprise deployment.
             log.info('Your documentation should be available shortly.')
+        elif page_type == 'user' or page_type == 'org':
+            username, repo = path.split('/', 1)
+            url = 'https://{}.github.io/'.format(username)
+            log.info('Your documentation should shortly be available at: ' + url)
```

Comment:
@waylan I understand what you're saying and I'll close the PR if it won't be merged. However, I believe these lines add value to `gh-deploy`, as I couldn't seem to get my own personal pages to deploy without being deployed to the URL https://mfjackson.github.io/mfjackson.github.io, which is not what I wanted. Is this the expected behavior or was I doing something wrong on my end?

I instead wanted them to be deployed to the URL https://mfjackson.github.io, which is what the `--page-type` option allows a user to specify. Initially I thought you could do this from the project repo, but I couldn't find a way to implement it without nuking the markdown files themselves.

Reply:
If that's all you're concerned about, then all we really need to do is fix the bug where the url is generated. But that url is only used in a message output to the user. It has no bearing on the actual deployment. And that's my point. This is a major change in the user facing interface only to fix a bug in an `info` message. Seems like more than is needed.

Now, if you can demonstrate an actual issue with the deployment. Show that without this change, things get deployed to GitHub incorrectly. If you can demonstrate that, then my view may be different.